DuckDuckGo/PrivacyProtectionOverviewController.swift

```diff
@@ -114,6 +130,7 @@ class PrivacyProtectionOverviewController: UITableViewController {
         updateTrackers()
         updatePrivacyPractices()
         footer.leaderboard.isHidden = true
+        updateFooterHeight()
```

Comment:
This was stupid on my part - I put this isHidden = true "fix" in for the leaderboard to try and fix the jump, but when the dashboard receives a subsequent update it was hiding it again.  This fixes that.